python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -66,12 +69,17 @@ def __init__(
                          or its expression must only depend on the
                          SlidingWindowTransform features specified earlier in this list
                          and their group-by keys.
+        :param timestamp_field: The name of the field whose values show the time when
+                                the corresponding sliding window row is generated.
+        :param timestamp_format: The format of the timestamp field.
         """
         super().__init__(
             name=name,
             source=source,
             features=features,
```

Comment:
Should we add to the `features` a new feature that represents the window end time generated by the SlidingFeatureView?

Otherwise, if we simply pass this timestamp field to FeatureView without adding a feature, we end up with a TableDescriptor whose `self.timestamp_field` can not be found in its `self.get_output_fields(...)`.

If this is indeed a bug, can you add a unit test to cover it in the latest PR?

Reply:
Good catch! It is indeed a bug. I add the window_time feature to the init method of SlidingFeatureView. Unit test is also added to cover the case.